WordPress/gutenberg
# RangeControl: Convert unit tests to typescript

Depends on:
- https://github.com/WordPress/gutenberg/pull/40535

Related:
- https://github.com/WordPress/gutenberg/issues/40507
- https://github.com/WordPress/gutenberg/pull/41444

## What?

Converts the `RangeControl` unit tests to typescript.

## Why?

Completes the process of converting the `RangeControl` to TypeScript alongside #40535 & #41444

## How?

- Coverts unit test file to TypeScript and resolve typing errors

## Testing Instructions

1. Run unit tests and ensure they are all passing: `npm run test-unit /packages/components/src/range-control/test`
